consensus/src/round_manager.rs

```diff
@@ -493,8 +493,6 @@ impl RoundManager {
         info!(
             self.new_log(LogEvent::ReceiveProposal).remote_peer(author),
             proposal = %proposal,
```

Comment:
You should be able to remove this, because the 2 things you were missing, are below as block_hash and block_parent_hash.

Reply:
woop, very good point!